var/spack/repos/builtin/packages/lbann/package.py

```diff
@@ -188,8 +188,8 @@ def cmake_args(self):
                 '-DConduit_DIR={0}'.format(spec['conduit'].prefix)])
 
         # Add support for OpenMP
-        if (self.spec.satisfies('%clang')):
-            if (sys.platform == 'darwin'):
+        if spec.satisfies('%clang') or spec.satisfies('%apple-clang'):
+            if sys.platform == 'darwin':
```

Comment:
I think what we want here is only `if spec.satisfies('%apple-clang'):`

Reply:
I think the same thing applies if someone is using an LLVM installed from brew or Spack on darwin.

Reply:
I spoke with @bvanessen yesterday on Slack. This code was to support LLVM installed with brew. I'm not sure about `apple-clang` (but we can fix it later) since it would basically require that people install `libomp` via e.g. brew like shown [here](https://iscinumpy.gitlab.io/post/omp-on-high-sierra/) and at that point I think the directories won't match.